components/tests/ui/robot_setup.sh

```diff
@@ -59,6 +60,19 @@ do
   bin/omero import -d $delDs $TINY_IMAGE_NAME --debug ERROR
 done
 
+# Create Screen with empty plates for Create Scenario
+scrDs=$(bin/omero obj new Screen name='CreateScenario')
+for (( k=1; k<=3; k++ ))
+do
+  bin/omero import -d $scrDs $PLATE_NAME --debug ERROR
```

Comment:
This command failed https://ci.openmicroscopy.org/view/Failing/job/OMERO-DEV-merge-robotframework/95/console